crates/rune-macros/src/any.rs

```diff
@@ -48,8 +48,20 @@ impl InternalCall {
             Ok(())
         };
 
+        #[cfg(feature = "dynamic_fields")]
+        let meta_fields = Some(quote! {Never});
+        #[cfg(not(feature = "dynamic_fields"))]
+        let meta_fields = None;
```

Comment:
Some additional simplifications in `rune-macros` would be possible when we go for the other feature gating strategy I'm proposing (see below). So I'm not gonna comment on them now ;).